packages/yourloops/components/patient-data.tsx

```diff
@@ -60,35 +61,48 @@ function PatientDataPageError({ msg }: PatientDataPageErrorProps): JSX.Element {
 
 function PatientDataPage(): JSX.Element | null {
   const paramHook = useParams();
+  const authHook = useAuth();
   const teamHook = useTeam();
   const dataHook = useData();
 
   const [patient, setPatient] = React.useState<Readonly<TeamUser> | null>(null);
   const [error, setError] = React.useState<string | null>(null);
 
   const { blipApi } = dataHook;
-  const { patientId } = paramHook as PatientDataParam;
+  const { patientId: paramPatientId = null } = paramHook as PatientDataParam;
+  const userId = authHook.user?.userid ?? null;
+
+  const initialized = authHook.initialized() && teamHook.initialized && blipApi !== null;
 
   React.useEffect(() => {
-    if (blipApi === null) {
+    log.debug("useEffect", { initialized, patient, userId, paramPatientId });
+    if (!initialized) {
       return;
     }
 
-    if (typeof patientId === "undefined") {
+    let patientId: string | null = null;
```

Comment:
you could use the Nullish Coalescing (??) 
`let patientId: string | null = paramPatientId ?? userId;`